docs/products/kafka/kafka-connect/howto/kafka-connect-debezium-tls-pg.md

```diff
@@ -0,0 +1,213 @@
+  correct by testing the database connection:
+
+  ```bash
+  psql "sslmode=verify-ca \
+    sslrootcert=server-ca.pem \
+    sslcert=client-cert.pem \
+    sslkey=client-key.pem \
+    hostaddr=<YOUR_IP_ADDRESS> \
+    port=5432 \
+    user=<YOUR_USER> \
+    dbname=<YOUR_DB_NAME>"
+  ```
+
+- **Set Debezium output plugin**: Use `pgoutput` or `decoderbufs` as the `plugin.name` in
+  your Debezium connector for logical decoding.
+
+    :::note
+      Starting with Debezium 2.5, the `wal2json` plugin is deprecated. You can use
+      either `pgoutput` or `decoderbufs` as the recommended replacement plugin.
+    :::
+
+## Variables
+
+The following table lists the variables for the configuration steps. Replace them with
+your actual environment values in the provided code snippets:
+
+| Variable | Description |
+|---|---|
+| `<kafka_cluster_name>` | Name of your Aiven for Apache Kafka service |
+| `<kafka_connect_name>` | Name of your Apache Kafka Connect service |
+| `<cloud_provider_and_region>` | Your cloud provider and region identifier |
+| `<plan>` | Service plan for Apache Kafka/Apache Kafka Connect |
+| `<project_name>` | Name of your project in the Aiven platform |
+| `<integration_config>` | JSON configuration for PostgreSQL endpoint |
+| `<integration_endpoint_id>` | ID of the created PostgreSQL endpoint |
+| `<connector_config>` | JSON configuration for Debezium connector |
+
+## ## Configure the integration
+
+1. Verify your existing Aiven for Apache Kafka service is active and accessible.
+   If you don’t have one, create an Apache Kafka cluster using this command:
+
+    ```bash
+    avn service create <kafka_cluster_name> \
+      --service-type kafka \
+      --cloud <cloud_provider_and_region> \
+      --plan <plan> \
+      --project <project_name>
+    ```
+
+    :::note
+    Ensure [topic auto-creation](/docs/products/kafka/howto/create-topics-automatically)
+    is enabled to automatically generate required Apache Kafka topics. If disabled,
+    manually create topics before starting the connector.
+    :::
+
+1. Create an Apache Kafka Connect service and configure it to communicate with your
+   Aiven for Apache Kafka service:
+
+    ```bash
+    avn service create <kafka_connect_name> \
+      --service-type kafka_connect \
+      --cloud <cloud_provider_and_region> \
+      --plan <plan> \
+      --project $PROJECT
+    ```
+
+1. Create an external PostgreSQL integration endpoint to represent your PostgreSQL
+   database. Use the following JSON configuration and replace the placeholders:
+
+    ```bash
+    INTEGRATION_CONFIG=$(cat <<-END
+    {
+      "ssl_mode": "verify-ca",
+      "host": "<postgresql_host>",
+      "port": <postgresql_port>,
+      "user": "<postgresql_user>",
+      "ssl_client_certificate": "$(cat /path/to/your/client-cert.pem)",
+      "ssl_root_cert": "$(cat /path/to/your/ca.pem)",
+      "ssl_client_key": "$(cat /path/to/your/client-key.pem)"
```

Comment:
we should add here also the `password`, the api has been changed and moved from the connector to here

Reply:
it should be another key value like: `"password": my_password`